customcommands/interval.go

```diff
@@ -118,7 +139,7 @@ func UpdateCommandNextRunTime(cc *models.CustomCommand, updateLastRun bool, clea
 		}
 	}
 
-	if cc.TriggerType != int(CommandTriggerInterval) || cc.TimeTriggerInterval < 1 {
+	if (cc.TriggerType == int(CommandTriggerInterval) && cc.TimeTriggerInterval < 1) || !(cc.TriggerType == int(CommandTriggerInterval) || cc.TriggerType == int(CommandTriggerCron)) {
```

Comment:
Personally, I think this conditional takes a bit of time to grok because of the nested operators. I think we can actually drop the first check and rewrite to
```go
if cc.TriggerType != int(CommandTriggerInterval) && cc.TriggerType != int(CommandTriggerCron) {
	return nil
}
```
since `cc.TimeTriggerInterval < 1` is handled by `CalcNextRunTime` anyway, and it's an edge case so an early exit isn't critical.
https://github.com/botlabs-gg/yagpdb/blob/ab2ea2d4de844e4b98198099f5aac38730ff911a/customcommands/interval.go#L29-L32

If you prefer to keep the check, it might nice to separate it into two conditionals or simplify the second condition to `cc.TriggerType != int(CommandTriggerInterval) && cc.TriggerType != int(CommandTriggerCron)`.